instrumentation/jvm-metrics/src/main/java/com/splunk/opentelemetry/instrumentation/jvmmetrics/otel/OtelJvmThreadMetrics.java

```diff
@@ -50,25 +50,10 @@ public void install() {
     Meter meter = OtelMeterProvider.get();
     ThreadMXBean threadBean = ManagementFactory.getThreadMXBean();
 
-    meter
-        .gaugeBuilder("runtime.jvm.threads.peak")
-        .setUnit("threads")
-        .setDescription(
-            "The peak live thread count since the Java virtual machine started or peak was reset.")
-        .buildWithCallback(measurement -> measurement.record(threadBean.getPeakThreadCount()));
-
-    meter
-        .gaugeBuilder("runtime.jvm.threads.daemon")
-        .setUnit("threads")
-        .setDescription("The current number of live daemon threads.")
-        .buildWithCallback(measurement -> measurement.record(threadBean.getDaemonThreadCount()));
-
-    meter
-        .gaugeBuilder("runtime.jvm.threads.live")
-        .setUnit("threads")
-        .setDescription(
-            "The current number of live threads including both daemon and non-daemon threads.")
-        .buildWithCallback(measurement -> measurement.record(threadBean.getThreadCount()));
+    // runtime.jvm.threads.peak was removed; easy to replace with
+    //   max(process.runtime.jvm.threads.count)
+    // runtime.jvm.threads.daemon is replaced by OTel process.runtime.jvm.threads.count{daemon=true}
+    // runtime.jvm.threads.live is replaced by OTel process.runtime.jvm.threads.count{daemon=false}
 
```

Comment:
There is one more meter `runtime.jvm.threads.states` in this class: there is currently no equivalent in OTel metrics, but there's an issue requesting this: https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/7006
I think we can leave it as it is for now, hopefully at some point it'll get implemented in the upstream